index.html

```diff
@@ -1709,53 +1793,47 @@
           partial interface Navigator {
             [SameObject] readonly attribute Presentation presentation;
           };
-
+        
 </pre>
         <p>
           The <dfn for="Navigator"><code>presentation</code></dfn> attribute is
           used to retrieve an instance of the <a>Presentation</a> interface.
         </p>
-        <pre class="idl">
-          interface Presentation {
-            // This API used by the controlling browsing context.
-            attribute PresentationRequest? defaultRequest;
-
-            // This API is available on the receiving browsing context.
-            [SameObject]
-            readonly attribute PresentationReceiver? receiver;
-          };
-
-</pre>
-        <p>
-          The <dfn for="Presentation"><code>defaultRequest</code></dfn> MUST
-          return the <a>default presentation request</a> if any,
-          <code>null</code> otherwise.
-        </p>
-        <p>
-          The <dfn for="Presentation"><code>receiver</code></dfn> MUST return
-          <code>null</code> if the current <a>browsing context</a> is not a
-          <a>receiving browsing context</a> and return a
-          <a>PresentationReceiver</a> instance otherwise.
-        </p>
         <section>
           <h4>
-            Setting the default presentation request
+            Controlling user agent
           </h4>
           <p>
+            The following partial interface MUST be implemented by a
+            <a>controlling user agent</a>:
+          </p>
+          <pre class="idl">
+            partial interface Presentation {
+              attribute PresentationRequest? defaultRequest;
+            };
```

Comment:
I started with that but the prose felt awkward, since there is clearly one part of the interface that applies only to a controlling UA and another that applies to a receiving UA.